client/modules/Layout/Layout.js

```diff
@@ -34,10 +34,10 @@ export class App extends Component {
         {this.state.isMounted && !window.devToolsExtension && process.env.NODE_ENV === 'development' && <DevTools />}
         <div>
           <Header />
-          <UserBar />
           <div className={styles.container}>
             {this.props.children}
           </div>
+          <UserBar />
```

Comment:
Why did you move the UserBar under the main content? It doesn't change the design itself, as it absolutely positioned, just want to understand your thinking here.